ConditionalFormatting/scripts/cf-rule-extraction/Dockerfile.linux

```diff
@@ -0,0 +1,16 @@
+# Build with: pipeline-tool docker-build --image-name excel-online-calc-test/commonimages/1-xlsx-to-json [--build-loc <this directory>]
```

Comment:
What is pipeline-tool? Is this perhaps a stale comment specifically for the internal setup?

Reply:
Yup. Removed.